### PR TITLE
#164798246 Update travis-ci to run tests

### PR DESCRIPTION
**Pull request by Genza999**
**What does this PR do?**
    

- Update TravisCI to run tests

**Description of the tasks to be completed?**

- Create a .travis.yml file to hold the Travis configurations.

- Enable Travis to run available tests

- Add Travis badge to README.md

**How should this be manually tested?**

- Click [here](https://travis-ci.org/andela/ah-frontend-prime) to view/trigger/restart a build

**Screenshots**
Screen Shot 2019-04-30 at 14 49 45

**What are the relevant pivotal tracker stories?**

- [#164798246](https://www.pivotaltracker.com/story/show/164798246)

### CODE REVIEW

**Review by e-ian — APPROVED** [review]
LGTM

**Review by patrickf949 — APPROVED** [review]
LGTM

**Review by anyatibrian — COMMENTED** [review]
great work @Genza999 on setting up Travis, though what am seeing is just the Travis badge and also you need to add the  coveralls badge also .in case you need some clue on how it's done  follow this `https://docs.travis-ci.com/user/coveralls/` to help you in setting up coverage for react

**Review by MuhanguziDavid — COMMENTED** [review]
Nice work @Genza999 , please go ahead to add the coveralls badge as well.

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ah-frontend-prime
+# Authors Haven - A Social platform for the creative at heart.
+
+[![Build Status](https://travis-ci.org/andela/ah-frontend-prime.svg?branch=ch-setup-travis-164798246)](https://travis-ci.org/andela/ah-frontend-prime)
```
Comment (MuhanguziDavid) [inline README.md]: Link the badge to the `develop` branch instead of the `ch-setup-travis-164798246` branch.

**Pull request merged**